app/src/main/java/com/example/myapplication/fragments/MyContactsFragment.kt

```diff
@@ -110,7 +146,12 @@ class MyContactsFragment : Fragment() {
             childFragmentManager,
             viewLifecycleOwner
         ) { name, career ->
-            viewModel.addUser(User(name, career, "", 0L))
+            viewModel.addUser(
+                User(
+                    name, career, "",
+                    ((viewModel.usersLiveData.value?.size)?.plus(1))!!.toLong(), false
```

Comment:
I suggest passing simple parameters to viewmodel, and build a UserModel there, should also help avoid accessing live data value directly